tldw_Server_API/app/core/DB_Management/ChaChaNotes_DB.py

```
@@ -16013,6 +16537,7 @@ def _deserialize_row_fields(self, row: sqlite3.Row, json_fields: list[str]) -> d
 
     _CHARACTER_CARD_JSON_FIELDS = ['alternate_greetings', 'tags', 'extensions']
     _STUDY_PACK_JSON_FIELDS = ['source_bundle_json', 'generation_options_json']
+    _SUGGESTION_SNAPSHOT_JSON_FIELDS = ['payload_json', 'user_selection_json']
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)